itou/siae_evaluations/models.py

```diff
@@ -203,26 +194,40 @@ def eligible_job_applications_under_ratio(self, evaluated_siaes):
             )
         ]
 
-    def populate_campaign(self):
-
+    def populate_campaign(self, now):
         if self.evaluations_asked_at:
-
             return f"La selection de l'échantillon à contrôler a déjà été réalisée pour {self.name}"
 
         with transaction.atomic():
             if not self.percent_set_at:
-                self.percent_set_at = timezone.now()
-            self.evaluations_asked_at = timezone.now()
+                self.percent_set_at = now
+            self.evaluations_asked_at = now
 
             self.save(update_fields=["percent_set_at", "evaluations_asked_at"])
 
             evaluated_siaes = EvaluatedSiae.objects.bulk_create(
                 EvaluatedSiae(evaluation_campaign=self, siae=Siae.objects.get(pk=pk))
                 for pk in self.eligible_siaes_under_ratio()
             )
-            EvaluatedJobApplication.objects.bulk_create(self.eligible_job_applications_under_ratio(evaluated_siaes))
+            EvaluatedJobApplication.objects.bulk_create(self.evaluated_job_applications(evaluated_siaes))
+
+            connection = mail.get_connection()
+            emails = [evaluated_siae.email_eligible_siae() for evaluated_siae in evaluated_siaes]
+            connection.send_messages(emails)
+
             return f"{len(evaluated_siaes)} SIAE(s) ont été ajoutées dans la campagne {self.name}"
 
+    # Emails.
+    def email_campaign_is_setup(self, ratio_selection_end_at):
+        to = self.institution.active_members
+        context = {
+            "ratio_selection_end_at": ratio_selection_end_at,
+            "dashboard_url": f"{settings.ITOU_PROTOCOL}://{settings.ITOU_FQDN}{reverse('dashboard:index')}",
+        }
+        subject = "siae_evaluations/email/campaign_is_setup_subject.txt"
+        body = "siae_evaluations/email/campaign_is_setup_body.txt"
```

Comment:
meme remarque que le nommage de la fonction mais sur le nom des fichiers de template: je ne comprends pas le `is` et ça ne laisse pas transparaitre de quel email du flow de campagnes d'eval on parle (il y en a ou aura plusieurs)